client/my-sites/plan-features/header.jsx

```diff
@@ -229,6 +233,18 @@ export class PlanFeaturesHeader extends Component {
 		}
 
 		if ( typeof discountPrice !== 'number' || typeof rawPrice !== 'number' ) {
+			if (
+				! isInSignup &&
+				! isMonthlyPlan &&
+				planMatches( planType, { group: GROUP_WPCOM, term: TERM_ANNUALLY } )
+			) {
+				if ( isBillingTimeframeVariation ) {
+					const price = formatCurrency( fullRawPrice, currencyCode, { stripZeros: true } );
+					return translate( 'Per month, %(price)s billed yearly', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( '%(price)s per month, billed yearly' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/billing-timeframe-experiment-annual-plans)